### ISSUE TITLE
Login fails on Android 6.0.1

### ISSUE THREAD

**Issue by pi3ab4** [tutao/tutanota#6939]
**Bug in mobile app**

**Describe the bug**
For a few weeks now, I have been unable to log into my account ("Add account") with the tutanota app on my smartphone. 
The message appears:
“The connection to the server was lost, please try again”.
a) Either this message appears immediately
b) or some status messages such as "Emails are being decrypted" etc. appear in quick succession, and at the end the above message appears.

Android 6.0.1
Release 227.240502.0

This problem does not occur with the desktop client when I log in to my account.

**To Reproduce**
1. Samsung Smartphone Galaxy S5 mini SM-G800F
2. Android 6.0.1
3. Install Release 227.240502.0
4. Try to log in
5. See error

**Expected behavior**
Log in works.

**Comment by CAberry** [comment]
Hello ! Bug Confirmed.

```
Feedback v227.240502.0 - CryptoError - PAID - android
Client: android
Type: PAID
Tutanota version: 227.240502.0
Timestamp (UTC): Tue, 14 May 2024 22:40:59 GMT
User agent:
Mozilla/5.0 (Linux; Android 14; 22101320G Build/UKQ1.231003.002; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/124.0.6367.123 Mobile Safari/537.36
CryptoError
Error message: mac expected but not present
Stacktrace:
CryptoError: mac expected but not present
at $s (https://assets.tutanota.com/worker.js:1:55205)
at xs (https://assets.tutanota.com/worker.js:1:55028)
at Fa (https://assets.tutanota.com/worker.js:7:46224)
at Co.unlockUserGroupKey (https://assets.tutanota.com/worker.js:18:51569)
at Qa.initSession (https://assets.tutanota.com/worker.js:18:23363)
at async Qa.finishResumeSession (https://assets.tutanota.com/worker.js:18:22640)
at async Qa.asyncResumeSession (https://assets.tutanota.com/worker.js:18:21833)
```

Please fix asap.

Greetings,

**Comment by tsaoi** [comment]
Bug confirmed on Android 13 running latest version of Tuta Version 229.240513.0

Can't even send a log as it is permanently offline. 


de.tutao.tutanota
versionCode 396340

targetSdk 33
minSdk 23

Installed: 6 Apr 2024; 08:48
Updated: 14 May 2024; 21:05

**Comment by shobeira** [comment]
The Android app has not been operational since yesterday and the app is unable to connect to the servers and even send the big report.

It is expected to be reliable and stable in professional grade services. It is not acceptable to have issues one day on the desktop app and another day on the mobile app!

**Comment by tsaoi** [comment]
@pi3ab4 can you please edit the Title of the issue to reflect Android overall? Not just affecting 6.0.1.

I wonder if any of you are on GrapheneOS, as I am but not sure if that is related to this bug.
Thanks.

**Comment by jomapp** [comment]
Thank you for reporting this issue!

Please update to the latest version 229.240514.1 which fixes this issue.

**Comment by pi3ab4** [comment]
@jomapp Thanks for taking care.

The bug is still not fixed in the latest version 229.240514.1.

Same behavior as described above (Android 6.0.1).

Please set status to "open".

Fun fact on the side.
The status messages go like this:
...
"Updating offline database..."
"Cycle to work..."

Original German:
"Update offline database..."
"Cycle to work..."

**Comment by clarkemcrobb** [comment]
Android app still refuses to login. I see this is closed, does that mean there is a fix?
